openqa_label_all_tests_with_one_failing_modules

```diff
@@ -60,14 +62,18 @@ def call(cmds, dry_run=False):
 
 
 def main():
+    request_params = {}
     args = parse_args()
     openqa_cmd = ['openqa-client', '--host', args.openqa_host, '--json-output']
     def openqa_call(cmds):
         return call(openqa_cmd + cmds, args.dry_run)
 
+    if args.groupid:
+        request_params.update({'groupid':args.groupid})
```

Comment:
no, as I stated, we need the additional space after the colon

Reply:
ups, was a detail I've overlooked.